### PR TITLE
Feat CTRL + Y to submit the solution

### LINKED ISSUE #1408: [Feature]: keyboard shortcut to submit  a solution
### Description

https://typehero.dev/challenge/default-generic-arguments
User should be able to submit the solution by pressing some keyboard combination. (CTRL + Y)
image

### Explanation

_No response_

### PR DESCRIPTION
**Pull request by sital002**
## Description

I have added a shortcut (CTRL + Y )to submit the solution.

## Related Issue

Closes #1408

### CODE REVIEW

File: packages/monaco/src/index.tsx [hunk 1]
```diff
@@ -101,6 +102,22 @@ export function CodePanel(props: CodePanelProps) {
   };
   const hasFailingTest = tsErrors?.some((e) => e.length) ?? false;
 
+  useEffect(() => {
+    const onSubmit = (e: KeyboardEvent) => {
+      if ((e.ctrlKey || e.metaKey) && e.code === 'KeyY') {
+        e.preventDefault();
+        if (submitButtonRef.current) {
```
Comment (bautistaaa) [inline packages/monaco/src/index.tsx:109]: you don’t need to simulate clicking the button. just call the same function

Comment (sital002) [reply packages/monaco/src/index.tsx:109]: I tried calling the handleSubmit funtion but when directly calling the handleSubmit the tsErros will be undefined  so every sumbit passed

Comment (bautistaaa) [reply packages/monaco/src/index.tsx:109]: hmm i’ll take a look

Comment (sital002) [reply packages/monaco/src/index.tsx]: quick update it's now working the linter didn't give me any warning regarding missing dependency.

**Comment by sital002** [comment]
@bautistaaa  can you review this PR.

**Review by bautistaaa — APPROVED** [review]

**Pull request merged**